packages/amplify-category-geo/package.json

```diff
@@ -22,9 +22,10 @@
     "aws"
   ],
   "dependencies": {
-    "@aws-cdk/aws-iam": "^1.159.0",
-    "@aws-cdk/aws-lambda": "^1.159.0",
-    "@aws-cdk/core": "^1.159.0",
+    "@aws-cdk/aws-iam": "^1.180.0",
+    "@aws-cdk/aws-lambda": "^1.180.0",
+    "@aws-cdk/aws-location": "^1.180.0",
+    "@aws-cdk/core": "^1.180.0",
```

Comment:
we should align this across packages including category api repo. @jhockett please correct me if I'm wrong.
does the feature require this particular version of cdk ?

Reply:
+1, if the newer cdk version is required, it should be bumped in all of our packages to match.

Reply:
Seeing these type errors when using v1.159.0.
<img width="1032" alt="SCR-20221107-n5y" src="https://user-images.githubusercontent.com/12038441/200445929-756ac2a2-99c1-4ec1-acd0-715d727b58bc.png">

Reply:
you may need to `yarn clean` , undo changes from `yarn.lock` or some combination of these. you can also inspect node_modules to see if you have mixed versions.

Reply:
Updated cdk dependencies across all packages to v1.180.0